test-suite/CMakeLists.txt

```diff
@@ -13,9 +17,8 @@ add_djinni_target(DjinniAllTests
   CPP_OUT "${CMAKE_CURRENT_BINARY_DIR}/generated-src/cpp"
   CPP_NAMESPACE "testsuite"
   IDENT_CPP_ENUM_TYPE "foo_bar"
-  CPP_OPTIONAL_TEMPLATE "std::experimental::optional"
-  CPP_OPTIONAL_HEADER "<experimental/optional>"
-  CPP_OPTIONAL_HEADER "\"test-suite/handwritten-src/cpp/optional.hpp\""
+  CPP_OPTIONAL_TEMPLATE "std::optional"
+  CPP_OPTIONAL_HEADER "\"optional\""
```

Comment:
In the python-branch this is set to `<optional>`. But we are on Windows and things are different here \*sigh\*: `<` or `>` break the djinni call on powershell/cmd. Putting escaped `"`  around `<optional>` does not seem to help.

As the choice of `<optional>` over `"optional"` is mostly aesthetics/convention, I decided the easiest fix is to but optional in `"`.